enterprise/lang-js/src/test/java/io/crate/operation/language/JavascriptUserDefinedFunctionTest.java

```diff
@@ -116,7 +117,24 @@ public void testInvalidJavascript() throws ScriptException{
         );
 
         String validation = udfService.getLanguage(JS).validate(udfMeta);
-        assertThat(validation, startsWith("Invalid JavaScript in function 'doc.f(double)': <eval>:1:27"));
+        assertThat(validation, startsWith("Invalid JavaScript in function 'doc.f(double)'"));
+        assertThat(validation, endsWith("^ in <eval> at line number 1 at column number 27"));
+    }
+
+    @Test
+    public void testInvalidJavascriptThrowsAssertion() {
```

Comment:
Would make this `testInvalidJavascriptThrowsAssertion*Error*() {` because I was confused at first how it can throw an assertion.

Reply:
Yes, you're right of course, it is a `java.lang.AssertionError`.